packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeErrorRenderer.tsx

```diff
@@ -3,78 +3,100 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-import { ComponentPropsWithoutRef } from "react";
+import { ComponentPropsWithoutRef, forwardRef, ForwardRefExoticComponent, RefAttributes } from "react";
 import { Anchor, Text, Tree } from "@itwin/itwinui-react/bricks";
 import { MAX_LIMIT_OVERRIDE } from "../internal/Utils.js";
 import { PresentationInfoNode } from "../TreeNode.js";
-import { useTree } from "../UseTree.js";
+import { HierarchyLevelDetails, useTree } from "../UseTree.js";
 import { FlatNode } from "./FlatTreeNode.js";
 import { useLocalizationContext } from "./LocalizationContext.js";
-import { TreeNodeRenderer } from "./TreeNodeRenderer.js";
+
+/** @alpha */
+export interface TreeErrorActionsProps {
+  /** A callback to reload a hierarchy level when an error occurs and `retry` button is clicked. */
+  reloadTree?: (options: { parentNodeId: string | undefined; state: "reset" }) => void;
+  /** Action to perform when the filter button is clicked for this node. */
+  onFilterClick?: (hierarchyLevelDetails: HierarchyLevelDetails) => void;
+}
 
 interface TreeErrorRendererOwnProps {
   node: FlatNode<PresentationInfoNode>;
 }
 
-type TreeErrorRendererProps = TreeErrorRendererOwnProps &
-  Pick<ComponentPropsWithoutRef<typeof TreeNodeRenderer>, "onFilterClick" | "reloadTree"> &
+type TreeErrorRendererProps = Pick<ComponentPropsWithoutRef<typeof Tree.Item>, "style"> &
```

Comment:
I don't think we need `Pick<ComponentPropsWithoutRef<typeof Tree.Item>, "style">` for just a `style` property. You can add it to `TreeErrorRendererOwnProps`

Reply:
Changed.